docs/index.html

```diff
@@ -4,58 +4,60 @@
 ds: 6
 static_dir: static
 versions:
-    alert-dialog: 2.1
-    badge: 1.2
-    breadcrumbs: 1.2
-    button: 1.6
-    carousel: 1.1
-    checkbox: 2.0
-    combobox: 2.2
-    confirm-dialog: 2.1
-    cta-button: 1.2
-    details: 1.2
-    drawer-dialog: 2.0
-    eek: 1.1
-    filter-button: 2.0
-    filter-menu: 2.0
-    filter-menu-button: 2.0
-    floating-label: 2.2
-    fullscreen-dialog: 2.1
-    icon: 1.3
-    icon-button: 1.6
-    infotip: 2.0
-    inline-notice: 2.2
-    lightbox-dialog: 2.1
-    less: 1.0
-    link: 2.0
-    listbox: 2.2
-    listbox-button: 2.2
-    menu: 2.3
-    menu-button: 2.3
-    page-grid: 1.0.0
-    page-notice: 3.0
-    pagination: 2.0
-    panel-dialog: 2.0
-    program-badge: 1.0
-    progress-bar: 3.1
-    progress-spinner: 3.1
-    progress-stepper: 4.0
-    radio: 2.0
-    section-notice: 3.0
-    section-title: 4.0
-    select: 2.2
-    signal: 1.0
-    snackbar-dialog: 2.0
-    split-button: 1.6
-    star-rating: 1.0
-    star-rating-select: 1.0
-    switch: 2.1
-    tabs: 2.1
-    textbox: 2.2
-    toast-dialog: 2.2
-    tooltip: 2.0
-    tourtip: 2.0
-    typography: 1.1
+alert-dialog: 2.1
```

Comment:
I'm curious why the indentation was removed here?

Reply:
Yes, please reset indentations. We should avoid unnecessary diffs.